pkg/load/create.go

```diff
@@ -60,20 +62,20 @@ func getLoadJobFromYaml() (*batchv1.Job, error) {
 	return loadJob.(*batchv1.Job), nil
 }
 
-func constructLoadJob(alluxio *alluxiov1alpha1.AlluxioCluster, load *alluxiov1alpha1.Load, loadJob *batchv1.Job) {
-	loadJob.Name = utils.GetLoadJobName(load.Name)
-	loadJob.Namespace = alluxio.Namespace
+func constructLoadJob(alluxio alluxiocluster.AlluxioClusterer, load Loader, loadJob *batchv1.Job) {
```

Comment:
```suggestion
func populateLoadJob(alluxio alluxiocluster.AlluxioClusterer, load Loader, loadJob *batchv1.Job) {
```

seems strange that a "construct" function doesn't return anything